.github/workflows/pr-checks.yml

```diff
@@ -40,6 +40,8 @@ jobs:
         image: ghcr.io/open-feature/flagd-testbed:latest
         ports:
           - 8013:8013
+        volumes:
```

Comment:
To be tested once https://github.com/open-feature/test-harness/pull/15 has been merged